src/main/groovy/org/pih/warehouse/importer/ProductSupplierExcelImporter.groovy

```diff
@@ -48,7 +48,7 @@ class ProductSupplierExcelImporter extends AbstractExcelImporter {
     ]
 
     static Map propertyMap = [
-            active                                : ([expectedType: ExpectedPropertyType.StringType, defaultValue: null]),
+            active                                : ([expectedType: ExpectedPropertyType.StringType, defaultValue: true]),
```

Comment:
My issue with this solution is that we've defined a default value in two places. I would prefer to define the default in one place, so I think using @BindUsing gets us closer to that outcome. However, I'm still trying to understand why this is even an issue, so please respond to my other question when you get a chance.

Reply:
@jmiranda This change comes from `ExcelImportService`.
We are calling the `convertColumnMapConfigManyRows` function from this service and this function calls `convertColumnMapOneRow` to get the value for a single row. This function iterates over the column map for that row and for each cell checks:
```
if (value == null || ''.equals(value)) { //cheking for null, because otherwise 0 value will fail here
        log.debug "Value for column $columnName row $rowIndex is null or empty String.  Was trying to find property $propertyName.  Skipping setting its value in param map"
        returnParams[propertyName] = propertyConfigurationMap?.get(propertyName)?.defaultValue
} else {
        anyNonNullValues = true
        returnParams[propertyName] = value
}
```
and because of the line following line, it is included in our params map.
```
returnParams[propertyName] = propertyConfigurationMap?.get(propertyName)?.defaultValue
```
In Grails 1 the same part of the code looks like this:
```
if (value==null || ''.equals(value)) { //cheking for null, because otherwise 0 value will fail here
    log.info "Value for column $columnName row $rowIndex is null or empty String.  Was trying to set $propertyName.  Skipping setting its value in param map"
} else {
    returnParams[propertyName] = value
}
```

Reply:
Thank you @alannadolny. That makes sense.